tests/testUtil.ts

```diff
@@ -66,7 +67,27 @@ export const createAPIGatewayEvent = ({
     return eventTemplate
 }
 
-export const createAPIGatewayContext = (): Context => ({
+export const createSqsEvent = (...arns: string[]): SQSEvent => ({
+    Records: arns.map((arn) => ({
+        eventSourceARN: arn,
+        messageId: "message_id",
+        receiptHandle: "receipt_handle",
+        body: "",
+        attributes: {
+            AWSTraceHeader: "",
+            ApproximateReceiveCount: "",
+            SentTimestamp: "",
+            SenderId: "",
+            ApproximateFirstReceiveTimestamp: "",
+        },
+        messageAttributes: {},
+        md5OfBody: "",
+        eventSource: "aws:sqs",
+        awsRegion: "",
+    })),
+})
+
+export const createApiGatewayContext = (): Context => ({
```

Comment:
`createAPIGatewayContext` seems more consistent with the rest of the project (uppercase abbreviations)